src/test/resources/subdir/another-sample.csv

```diff
@@ -0,0 +1,10 @@
+4,IN,IN,0.0,0.0,2011-04-01,2011-04-30,all,1500217.7370499999,3370420.2702702703,1731201.75,1764537.27,156,445122.22542332584,2011-05-11,1,1
```

Comment:
There used to be a sample credentials file here. These resource files are used in integration tests in s3-test.clj namespace. 

Since credential files are going away, I replaced it with this one.